sdk/nodejs/runtime/settings.ts

```diff
@@ -63,6 +64,7 @@ const nodeEnvKeys = {
     monitorAddr: "PULUMI_NODEJS_MONITOR",
     engineAddr: "PULUMI_NODEJS_ENGINE",
     syncDir: "PULUMI_NODEJS_SYNC",
+    cacheDynamicProviders: "PULUMI_NODEJS_CACHE_DYNAMIC_PROVIDERS",
```

Comment:
I think the rest of the things in the list are automatically passed from the CLI.  This one seems different, as I don't think we expect the CLI to pass it.  Not sure that "matters" - but maybe at least a comment that this is different than the others and that users must set it themselves if they want to enable this?

Reply:
I don't think it matters, other keys such as `pulumiEnvs` below are controlled via individual environment variables. I'll add a comment to clarify